sonar-home/src/main/java/org/sonar/home/cache/FileHashes.java

```diff
@@ -58,7 +56,13 @@ public String of(InputStream input) {
       throw new IllegalStateException("Fail to compute hash", e);
 
     } finally {
-      IOUtils.closeQuietly(input);
+      if (input != null) {
+        try {
+          input.close();
+        } catch (IOException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)